google/cloud/aiplatform/featurestore/entity_type.py

```diff
@@ -1260,8 +1271,20 @@ def ingest_from_df(
         temp_bq_dataset = bigquery_client.create_dataset(temp_bq_dataset)
 
         try:
+
+            parquet_options = bigquery.format_options.ParquetOptions()
+            parquet_options.enable_list_inference = True
+
+            job_config = bigquery.LoadJobConfig(
+                schema=bq_schema,
+                source_format=bigquery.SourceFormat.PARQUET,
+                parquet_options=parquet_options,
+            )
+
             job = bigquery_client.load_table_from_dataframe(
```

Comment:
It looks like there's no assertion that the current args are passed to `bq_load_table_from_dataframe_mock`. Perhaps add to it to `test_ingest_from_df_using_column` and `test_ingest_from_df_using_datetime`.